src/AppBundle/Resources/views/Report/ProfDeputyCostsEstimate/breakdown.html.twig

```diff
@@ -19,11 +19,24 @@
 
 {% block pageContent %}
 
+    {{ form_start(form, {attr: {novalidate: 'novalidate'}}) }}
+
     <p class="heading-small push-half--bottom  text">
-        {{ (page ~ '.pageSectionDescription') | trans(transOptions) }}
+        {{ (page ~ '.form.profDeputyCostsEstimateManagementCost.sectionDescription') | trans(transOptions) }}
```

Comment:
You can just change the pageSectionDescription to be what you need it to be. Otherwise we'll have translation keys that are never used